TSOClient/tso.simantics/Primitives/VMSetToNext.cs

```diff
@@ -92,6 +92,20 @@ public override VMPrimitiveExitCode Execute(VMStackFrame context, VMPrimitiveOpe
                         entities = context.VM.Context.ObjectQueries.GetObjectsByGUID(operand.GUID); break;
                     case VMSetToNextSearchType.ObjectWithCategoryEqualToSP0:
                         entities = context.VM.Context.ObjectQueries.GetObjectsByCategory(context.Args[0]); break;
+                    case VMSetToNextSearchType.FSOObjectOfSemiGlobal:
+                        var sg = FSO.Content.Content.Get().WorldObjects.Get(operand.GUID).Resource.SemiGlobal;
+
+                        if (sg != null)
+                        {
+                            string sg_name = sg.Iff.Filename;   //retrieve semiglobal's iff filename
```

Comment:
Too many spaces before comment

Reply:
Solved in fc27d4be1bda305e6d4dc490743be9e8b6c21285